### PR TITLE
Compatibility fixes and tweaks

### PR DESCRIPTION
**Pull request by bennothommo**
This commit will fix https://github.com/rainlab/builder-plugin/issues/295 and https://github.com/rainlab/builder-plugin/issues/328.

Commit f1f0f4523d5d084779a18d481bfe3a615e4ff1f9 brought in changes that allowed null default values to be specified as `null` (without quotes) to be marked as nulled by default, with a quoted null default value to be a string value of `"null"`. It was reported by several people on Slack as well as #295 that this did not work. This rolls back the changes for null, while keeping the changes for the boolean default values. Doctrine, by default, will mark any column that is nullable as `null` by default, unless a specific default value is given.

This also brings in the https://github.com/marktopper/doctrine-dbal-timestamp-type library to provide support for timestamp columns.

Finally, some sensible widths were put in to columns of the database table datatable widget, to make it a bit more uniform.

As this is technically a breaking change, it will be released as a "critical" update.

### CODE REVIEW

File: Plugin.php [hunk 1]
```diff
@@ -153,5 +153,10 @@ public function register()
         CombineAssets::registerCallback(function ($combiner) {
             $combiner->registerBundle('$/rainlab/builder/assets/js/build.js');
         });
+
+        /**
+         * Add provider for "timestamp" type for MySQL and SQLite
+         */
+        $this->app->register(\MarkTopper\DoctrineDBALTimestampType\Laravel5ServiceProvider::class);
```
Comment (LukeTowers) [inline Plugin.php:160]: Is this going to break when L6 is released?

Comment (bennothommo) [reply Plugin.php:160]: @LukeTowers I don't think so - the timestamp field still isn't available in Doctrine with L6, but I'll give it a test to make sure.

Comment (LukeTowers) [reply Plugin.php:160]: I moreso meant that we're referencing a service provider specifically made for Laravel 5 so I'd be concerned about composer versions of the package no longer including it for a Laravel 6 install.

Comment (bennothommo) [reply Plugin.php:160]: @LukeTowers I've tested this with L6 and it works fine. The `composer.json` for this library does not specify a Laravel version, it only requires `doctrine/dbal`, and its restriction is compatible with Laravel 6. The service provider also only uses methods from Doctrine and does not interact with Laravel at all beyond latching to its service provider boot process.

**Comment by daftspunk** [comment]
Leaving this open to review prior change

**Comment by bennothommo** [comment]
Not when you're throwing shade on the commit messages, m8.

**Pull request closed without merging**